doc/source/user_guide/UploadToGallery.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Quick Start (oda api v1.1.26)"
+    "# Quick Start (oda api v1.1.28)"
```

Comment:
also please add it there https://github.com/oda-hub/oda_api/blob/d2426c71c141c0cc0d37b4adcf538986babcfe4c/setup.cfg#L21 so it updates automatically